### PR TITLE
upgrade subprocess.run(universal_newlines=True) to subprocess.run(text=True) in --py37-plus

### LINKED ISSUE #336: upgrade subprocess.run(universal_newlines=True) to subprocess.run(text=True) in --py37-plus

### PR DESCRIPTION
**Pull request by MarcoGorelli**
closes #336

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -536,3 +536,14 @@ Availability:
 -def f(x: 'queue.Queue[int]') -> C:
 +def f(x: queue.Queue[int]) -> C:
 ```
+
+
+### `subprocess.run`: replace `universal_newlines` with `text`
```
Comment (asottile) [inline README.md:541]: this is out of order

File: pyupgrade/_plugins/universal_newlines_to_text.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+def _replace_universal_newlines_with_text(
+    i: int,
+    tokens: List[Token],
+    *,
+    arg_idx: int,
+) -> None:
+    j = find_open_paren(tokens, i)
+    func_args, _ = parse_call_args(tokens, j)
+    src = tokens_to_src(tokens[slice(*func_args[arg_idx])])
+    new_src = src.replace('universal_newlines', 'text')
```
Comment (asottile) [inline pyupgrade/_plugins/universal_newlines_to_text.py:30]: this potentially breaks code if you write `universal_newlines=universal_newlines` -- this should just replace the one argument token

File: pyupgrade/_plugins/universal_newlines_to_text.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+) -> Iterable[Tuple[Offset, TokenFunc]]:
+    if (
+            state.settings.min_version >= (3, 7) and
+            is_name_attr(
+                node.func,
+                state.from_imports,
+                'subprocess',
+                ('run',),
+            ) and
+            not has_starargs(node)
```
Comment (asottile) [inline pyupgrade/_plugins/universal_newlines_to_text.py:48]: is there actually a problem with starargs here?

Comment (MarcoGorelli) [reply pyupgrade/_plugins/universal_newlines_to_text.py]: indeed, there's not - have removed and added a test!

**Review by asottile — APPROVED** [review]

**Pull request merged**